test/integ/quote-gouda.test.ts

```diff
@@ -25,6 +25,7 @@ import { NATIVE_ADDRESS, RoutingType } from '../../lib/constants';
 import { ClassicQuoteDataJSON, QuoteRequestBodyJSON, RoutingConfigJSON } from '../../lib/entities';
```

Comment:
Meanwhile we are here, I noticed that the integ-test has separate test setups against classic and dutch quotes, but not combined. We have unit test against [getBestQuote](https://github.com/Uniswap/unified-routing-api/blob/main/lib/handlers/quote/handler.ts#L354), but not the unilateral integ-test case to assert against classic through routing-api against the fillers (cc @zhongeric @rileydcampbell to confirm).

The best would be to add the integ-test to make sure the quote portion adjusted from classic and dutch are compared, because otherwise one might unfair advantage over another.